src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -566,14 +566,14 @@ CharMap =
 #    3gp      3rd Generation Partnership Project            video/3gpp
 #    3g2a     3rd Generation Partnership Project 2 Audio    audio/3gpp2
 #    3g2      3rd Generation Partnership Project 2          video/3gpp2
-#    aac-lc   Advanced Audio Coding (Low Complexity)        audio/aac
+#    aac-lc   Advanced Audio Coding (Low Complexity)        audio/mp4
```

Comment:
If you thing that callin them "MPEG-4 Part 3" or "MPEG-4 AAC" will help the users...good luck.